.circleci/config.yml

```diff
@@ -39,7 +34,7 @@ jobs:
       - run:
           name: Set NPM auth token
           command: |
-            npm config set //registry.npmjs.org/:_authToken "${NPM_TOKEN}"
+            echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
```

Comment:
I think this is "less right" than the previous version, but I know both should work so it doesn't matter too much. I suspect `_authToken` is deprecated in npm, but it should still work for now.